Formula/libyang.rb

```diff
@@ -0,0 +1,22 @@
+class Libyang < Formula
+  desc "YANG data modelling language parser, toolkit, and API written in C"
+  homepage "https://github.com/CESNET/libyang"
+  url "https://github.com/CESNET/libyang/", :using => :git, :tag => "v1.0-r2", :revision => "347246611b85e05d16f54faaa5697c4b2ee4b468"
```

Comment:
if you use https://github.com/CESNET/libyang.git you don't have to define the strategy afaik.